tests/nlu/featurizers/test_lm_featurizer.py

```diff
@@ -361,6 +361,7 @@ def evaluate_message_shapes(
             assert intent_sentence_vec is None
 
     @pytest.mark.timeout(120, func_only=True)
+    @pytest.mark.skip_on_windows
```

Comment:
Please add a Windows skip marker for `test_lm_featurizer_num_sub_tokens_process_training_data` as well.